src/components/Person/Person.jsx

```diff
@@ -1 +1,32 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age
+      && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {!isMarried
+          ? `I am not married`
+          : `${partnerName} is my ${sex === 'm'
+            ? `wife`
+            : `husband`}`}
```

Comment:
nested ternary is not a good idea, to determine wife or husband you can move it into variable and here just use this variable